WorkloadManagementSystem/Client/CPUNormalization.py

```diff
@@ -22,28 +22,53 @@
 UNITS = { 'HS06': 1. , 'SI00': 1. / 250. }
 
 def getMachineFeatures():
+  """ This uses the _old_ MJF information """
   features = {}
-  if 'MACHINEFEATURES' not in os.environ:
+  featuresDir = os.environ.get( "MACHINEFEATURES" )
+  if featuresDir is None:
     return features
   for item in ( 'hs06', 'jobslots', 'log_cores', 'phys_cores' ):
-    fname = os.path.join( os.environ['MACHINEFEATURES'], item )
+    fname = os.path.join( featuresDir, item )
     try:
       val = urllib.urlopen( fname ).read()
     except:
       val = 0
     features[item] = val
   return features
 
+def getJobFeatures():
+  """ This uses the _new_ MJF information """
+  features = {}
+  featuresDir = os.environ.get( "JOBFEATURES" )
+  if featuresDir is None:
+    return features
+  for item in ( 'hs06_job', 'allocated_cpu' ):
+    fname = os.path.join( featuresDir, item )
+    try:
+      val = urllib.urlopen( fname ).read()
+    except:
+      val = 0
+    features[item] = val
+  return features
+
+
 def getPowerFromMJF():
-  features = getMachineFeatures()
-  totalPower = features.get( 'hs06' )
-  logCores = float( features.get( 'log_cores', 0 ) )
-  physCores = float( features.get( 'phys_cores', 0 ) )
-  jobSlots = float( features.get( 'jobslots', 0 ) )
-  denom = min( max( logCores, physCores ), jobSlots ) if ( logCores or physCores ) and jobSlots else None
-  if totalPower and denom:
-    return round( float( totalPower ) / denom , 2 )
-  else:
+  try:
+    features = getJobFeatures()
+    if 'hs06_job' in features:
+      return round( float( features['hs06_job'] ), 2 )
+    features = getMachineFeatures()
+    totalPower = float( features.get( 'hs06', 0 ) )
+    logCores = float( features.get( 'log_cores', 0 ) )
+    physCores = float( features.get( 'phys_cores', 0 ) )
+    jobSlots = float( features.get( 'jobslots', 0 ) )
+    denom = min( max( logCores, physCores ), jobSlots ) if ( logCores or physCores ) and jobSlots else None
+    if totalPower and denom:
+      return round( totalPower / denom , 2 )
+    else:
+      return None
+  except Exception as e:
```

Comment:
In that you can just catch `ValueError`.

Another point is: if there is a mistake, being here "protected", printout or not, we would end up never noticing and so never notifying the site. I am wondering if we can do better. 

Yet another point, and certainly not for this PR: I don't think any other component that is not the pilot, in DIRAC, use the CPUNormalization. Maybe we can organize this slightly better.